src/main/java/com/autotune/common/datasource/DataSourceCollection.java

```diff
@@ -144,6 +144,10 @@ public void addDataSourcesFromConfigFile(String configFileName) {
                     DataSourceInfo dataSourceInfo = new ExperimentDBService().loadDataSourceFromDBByName(name);
                     if (null != dataSourceInfo) {
                         LOGGER.error("Datasource: {} already exists!", name);
+                        // add the auth details to local object
```

Comment:
Is this change  related to PR ?

Reply:
No, this was pulled from #1358 . It needs to be merged first.